tests/smoke-tests/test-balances-consistency.ts

```diff
@@ -116,6 +116,12 @@ describeSmokeSuite(`Verifying balances consistency...`, (context) => {
         : [],
     ]);
 
+    const networkName = context.polkadotApi.runtimeChain.toString();
+    let preimages = [];
+    if (networkName === "Moonbase Stage") {
+      preimages = await apiAt.query.preimage.statusFor.entries();
```

Comment:
This should be retrieved with the other elements too using a inline condition easily include it in the array